loadbalancer/src/main/java/com/io/loadbalancer/consistent_hashing/ImageManagerMonitor.java

```diff
@@ -0,0 +1,17 @@
+package com.io.loadbalancer.consistent_hashing;
+
+import java.util.stream.IntStream;
+
+public class ImageManagerMonitor {
+
+    int IMMax;
+
+    public ImageManagerMonitor(int IMMax) {
+        this.IMMax = IMMax;
+    }
+
+    public int[] getValidInstances(){
```

Comment:
This class should in the future be probably responsible for monitoring liveness of image managers. We can probably return a list of valid image managers instead of their indexes. This does not elegant if we purely operate on indexes (at some point you probably have to but keep operating on lists as much as possible).

Reply:
Changed function name so it is not misleading but kept operating on indexes as discussed.